README.md

```diff
@@ -1 +1,70 @@
-# react-initialization
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+#  toDoList-react
+
+> math magicians is a simple react caculator website to practice react .
+
+
+  practice react with building to-do list app module 3 week 2 :
+
+simple to-do list using React
+
+
+
+## Built With
+
+- HTML
+- CSS
```

Comment:
- [ ] **Optional** You have documented well your project 🟢 . I kindly suggest you it also be better if you share us the Live URL

Reply:
unfortunately, I have issues making a live demo for it but I will try to do it later. Thanks for your note. 😄